### ISSUE TITLE
[BUG] "frametime_color" config setting doesn't work properly anymore in 0.7.1

### ISSUE THREAD

**Issue by Perkolator** [flightlessmango/MangoHud#1260]
`frametime_color` used to work in previous versions, bug started in version 0.7.1. The frametime graph is now always in green, despite what the `frametime_color` setting is set to in config.

- Linux Mint 20.3 Cinnamon
- MangoHud 0.7.1
- RTX2060 Mobile

1. Set `frametime_color` in config, e.g `F87F00`.
2. Start game
= Frametime graph color is still default green.

**Expected behavior**
Frametime graph color should be changed too, like in the previous versions.

Here's few screenshots for comparison:

**0.7.0**:

![mangohud_0 7 0_minimum](https://github.com/flightlessmango/MangoHud/assets/53913724/b8961a38-e59c-45c7-8464-cd4a721901e7)
![mangohud_0 7 0_extended](https://github.com/flightlessmango/MangoHud/assets/53913724/8c5e785a-07bd-4330-b930-019ae7289041)

**0.7.1**:

![mangohud_0 7 1_minimum](https://github.com/flightlessmango/MangoHud/assets/53913724/e25bf5cf-6b0e-42d3-9f55-55f4d7d14e5d)
![mangohud_0 7 1_extended](https://github.com/flightlessmango/MangoHud/assets/53913724/e1b3ef07-6dee-403a-a692-c529efbc2ac6)

Here's the config file I'm using:

```
fps_limit=0,60

preset=-1,4

time=0

version=0

gpu_temp
gpu_core_clock
gpu_power=0
gpu_load_change
gpu_load_value=85,95
gpu_load_color=F87F00,F87F00,FF0001
gpu_fan=0
gpu_voltage=0

cpu_temp
cpu_power=0
cpu_mhz
cpu_load_change
cpu_load_value=85,95
cpu_load_color=F87F00,F87F00,FF0001

core_bars

vram
ram

fps_color_change
fps_value=55,59
fps_color=FF0001,FF0001,F87F00
frametime=0

font_size=22

text_outline_thickness=1.0

hud_no_margin

width=325
table_columns=4

background_alpha=0.0

text_color=F87F00
frametime_color=F87F00

toggle_fps_limit=Shift_R+F9
```

[Presets file can't be read when using MangoHud with Bottles](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.MangoHud/issues/38), so I came up with this single config file that enables me to switch between 2 different MangoHud setups, a minimal setup and an "extended" setup with more info (can't control every aspect with this "one config" trick, so the "extended" setup shows few details that I actually would like to hide but can't.. so this is a "best I can get" without using presets).

***

Additionally, what's with the change of the frametime graph to a bigger one with some "scale" background? (seen in the last screenshot) I didn't see anything related to that change in the 0.7.1 changelog. How can I get back the old frametime graph?

**Comment by flightlessmango** [comment]
> How can I get back the old frametime graph?

It's part of preset 4 and you can disable it with `frame_timing_detailed=0`

**Comment by flightlessmango** [comment]
Changing frametime color works as expected on my end, could you compile latest and see if it's been fixed already?

**Comment by Perkolator** [comment]
> Changing frametime color works as expected on my end, could you compile latest and see if it's been fixed already?

Could you test the actual 0.7.1 release yourself since that's the latest stable you have provided for users?

**Comment by zastrixarundell** [comment]
It doesn't work for me as well. When I change to histogram it works but not with graph.
![image](https://github.com/flightlessmango/MangoHud/assets/30195425/6df420ed-bff2-4dae-ab80-4dae264e5e13)
![image](https://github.com/flightlessmango/MangoHud/assets/30195425/fd110d27-5204-4b88-8d2f-7148fcc1919f)

**Comment by flightlessmango** [comment]
This was fixed bd37645 and will be included in next release
